File: FurnitureAPI/Models/AddFurnitureResultModel.cs
namespace FurnitureAPI.Models
{
    public class AddFurnitureResultModel
    {
        public int Id { get; set; }

        public string Name { get; set; }

        public float Price { get; set; }

        public int FurnitureTypeId { get; set; }
        public string FurnitureTypeName { get; set; }

        public string Description { get; set; }

        public string Size { get; set; }

        public int AvailableQuantity { get; set; }
        public string ImageUrl { get; set; }

    }
}

File: FurnitureAPP/app/src/main/java/com/example/furnitureapp/datalayers/retrofit/models/AddFurnitureResultModel.kt
package com.example.furnitureapp.datalayers.retrofit.models

import com.google.gson.annotations.SerializedName

data class AddFurnitureResultModel(
    @SerializedName("id")
    val id: Int,
    @SerializedName("name")
    val name: String,
    @SerializedName("imageUrl")
    val imageUrl: String,
    @SerializedName("price")
    val price: Float,
    @SerializedName("furnitureTypeId")
    val furnitureTypeId: Int,
    @SerializedName("furnitureTypeName")
    val furnitureTypeName: String,
    @SerializedName("description")
    val description: String,
    @SerializedName("size")
    val size: String,
    @SerializedName("availableQuantity")
    val availableQuantity: Int,
)